luigi/lock.py

```diff
@@ -33,9 +33,12 @@ def getpcmd(pid):
 
     :param pid:
     """
-    cmd = 'ps -p %s -o command=' % (pid,)
+    cmd = 'ps -o pid,comm | grep %s | xargs -n 2' % (pid,)
```

Comment:
Unit test just failed on travis looking into it right now. It was working just fine on the mac in python 3.5, and tested on the command line in other distros to make sure they had the needed tools by default.